src/nrnpython/nrnpy_hoc.cpp

```diff
@@ -2623,10 +2624,10 @@ extern "C" PyObject* get_plotshape_data(PyObject* sp) {
     void* that = pho->ho_->u.this_pointer;
 #if HAVE_IV
     IFGUI
-    spi = ((ShapePlot*) that);
+        spi = ((ShapePlot*) that);
 }
 else {
-    spi = ((ShapePlotData*) that);
+        spi = ((ShapePlotData*) that);
     ENDGUI
```

Comment:
Just a nitpick. Pramod's suggestion for the master for this identical situation seems reasonable to employ here. i.e.
```
// clang-format off
        IFGUI
            spi = ((ShapePlot*) that);
        } else {
            spi = ((ShapePlotData*) that);
        ENDGUI
// clang-format on
```

Reply:
That's just formatting, right?

Committed and pushed.

Reply:
Looks like it broke the formatting job, so I undid the commit again.

Reply:
I copy-pasted the function from the master branch. That appeased the formatting checker, so I'm resolving this.